tests/test_notebooks.py

```diff
@@ -43,6 +45,22 @@ def test_eln_import(selenium_driver):
     driver.find_element(By.ID, "tooltip")
 
 
-def test_computational_resources(selenium_driver):
-    driver = selenium_driver("notebooks/computational_resources.ipynb")
+def test_computational_resources(notebook_service, selenium):
+    """Test the quicksetup of the code"""
+    url, token, docker_compose = notebook_service
+
+    # install the aiida-quantumespresso, we need it for test of some widgets
+    install_command = "bash -c 'pip install -U aiida-quantumespresso==4.1.0'"
+    command = f"exec --workdir /home/jovyan/apps/aiidalab-widgets-base -T aiidalab {install_command}"
```

Comment:
I would use the `aiidalab_exec` fixture from aiidalab-docker-stack 

https://github.com/aiidalab/aiidalab-docker-stack/blob/e89338181ada44420051e775645fdea00f91cff3/tests/conftest.py#L46

Reply:
Yes, I think that makes things easier. Do you know the different between using `bash -c` and not use it? I am confused about it and think we don't need it in all the case here. But why it is needed for `"bash -c 'echo \"${PATH}\"'"` in https://github.com/aiidalab/aiidalab-docker-stack/blob/e89338181ada44420051e775645fdea00f91cff3/tests/test_aiidalab.py#L139?

Reply:
I think it is needed for escaping. Otherwise the `echo $PATH` does not work because the $PATH is not expanded.

Reply:
Makes sense! I'll try here removing all `bash -c`, it should work if you're correct about this.